### DISCUSSION TITLE
What is the default value of plus-emph-style?

### DISCUSSION THREAD

**Discussion by anakimluke** [dandavison/delta#1302] [Q&A] [answered]
I see the screenshots in readme have the theme the way I want, but when I run delta the background of added lines have a very dark, desaturated background. The default value of `plus-emph-style` I get is `syntax "#003300"`.

Are the screenshots using a custom theme?

I could/will change each color manually but I'm curious what's going on here.

**Comment by anakimluke** [comment, marked as answer]
welp, I found the default value in the manual

**Reply by anakimluke** [reply]
plus it was not `plus-emph-style` that was the problem, it was the other `plus-*`

**Reply by anakimluke** [reply]
also I learned I can't delete a discussion thread. feel free to do so if you can

**Comment by dandavison** [comment]
Hi @anakimluke, try `delta --show-config`. It prints out all the settings that are in effect and shows you colors etc.
